blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/FlowNodeWrapper.java

```diff
@@ -166,6 +174,17 @@ String blockError(){
         return null;
     }
 
+    boolean isBlockErrorInterruptedWithAbort() {
+        if (hasBlockError()) {
+            Throwable error = blockErrorAction.getError();
+            if (error instanceof FlowInterruptedException) {
```

Comment:
My recollection of `FlowInterruptedException`, it gets thrown not just in explicit ABORT but also in case of syntax error or other unexpected error. Maybe thats alright to call AOBRTED in such cases?